tools/hangar/src/generate_tests.ts

```diff
@@ -11,7 +11,16 @@ for (const fileInfo of readdirSync(validTestDir, { withFileTypes: true })) {
   }
 
   const filename = fileInfo.name;
-  const skipText = filename.endsWith("skip.w") ? ".skip" : "";
+  const skip = filename.endsWith("skip.w");
+
+  // TODO: Need to figure out what to do about docker-based tests on non-linux CI
+  const skipText =
+    skip ||
+    (process.env.CI &&
+      process.platform !== "linux" &&
+      basename(filename) === "redis.w")
+      ? ".skip"
+      : "";
```

Comment:
(random idea) WDYT of adding a convention where we just add comments to the head of the Wing example files? e.g.

```
// skip-mac skip-windows
bring cloud;

...
```

Reply:
I was working on introducing something like this for the benchmarking PR. Test metadata stuff. I do really want this, but since this issue is blocking the main build I figured a super quick patch is ok.